week2/2-website/css/style.css

```
@@ -16,4 +24,230 @@ body {
+   */
+
+ img.logo {
+    height: 7rem;
+    width: 2.6rem;;
+}
+
+nav.menu  {
+   /*  float: right; */
+    color: #838994;
+    font-size: 1.664rem;
+    /*
+    margin-top: 1.7rem;
+    margin-top: 2.752rem;
+    */
+}
+
+nav.menu li {
+    display: inline-block;
+    margin-left: 2.5008rem;
+}
+
+nav.menu .selected{
+    color: #4c5058;
+    font-weight: 500;
+}
+
+
+/*
+ *   Container for background photo, title and tagline
+ */
+
+.tag-container{
+    background-image: url('../img/first-background.jpg'); 
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 73.5008rem;
+}
+
+/*
+* Lets layout out content using ye ole trust flexbox.....
+*/
+.tag-content{
+    padding:0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+
+.tag-content h1{
+    margin: 0;
+    margin-top: 28rem;
+    font-size: 5rem;
+}
+
+.tag-content h3{
+    font-size: 2.5008rem;
+    margin-top: 2rem;
+}
+
+.tag-content h1, .tag-content h3 {
+    color: #fff;
+    font-weight: 300;
+}
+
+.tag-content button{
+    margin-top: 2.5008rem;
+    width: 15.5008rem;
+    background-color: #f15a29;
+    border-color: #f15a29;
+    text-shadow: none;
+    border-style: none;
+    border-radius: 0.4rem;
+    padding: 1.6rem 0.9rem;
+    color: white;
+    font-size: 1.8rem;
+    box-sizing: border-box;
+}
+
+.benefits-container{
+    display: flex;
+    justify-content: space-around;
+    text-align: center;
+    padding-bottom: 8rem;
+}
+
+.benefit img{
+    width: 12.7rem;
+    height: 12.7rem;
+}
+
+#benefits{
+    margin-top: 12.6rem;
+
+}
+#benefits h2{
+    margin-top: 12.6rem;
+    font-size: 4.4rem;
+    font-weight: 300;
+    margin: 2.6rem 0 5rem;
+    text-align: center;
+}
+.benefit h3{
+    font-size: 2.5008rem;
+    font-weight: 400;
+    margin: 4rem 0 3rem;
+}
+section.benefit{
+    width: 33.3333333%;
+}
+
+footer{
+    padding-bottom: 15rem;
+}
+
+footer hr{ 
+    margin: 2.1008rem;
+    border-top: 1px solid #eaebec;
+}
+
+#social-media-container h5{
+    margin: 2.1008;
+}
+#social-media-container h5{
+    font-size: 1.6rem;
+    font-weight: 400;
+    text-align: center;
+}
+.social-media-layout{
+    display: flex;
+    justify-content: center;
+}
+
+.social-media{
+    border-radius: 2rem;
+    width: 4rem;
+    height: 4rem;
+    margin: 2rem 5px;
+    border: 1px solid #eaebec;
+    padding: 1rem 0;
+    text-align: center;
+    box-sizing: border-box;
+}
+
+#social-media-container .twitter {
+    color: #55acee;
+}
+#social-media-container .facebook {
+    color: #4c66a4;
+}
+#social-media-container .instagram {
+    color: #3f729b;
+}
+
+.fa{
+    font-family: FontAwesome;
+    font-size: 1.4rem;
+}
+
+#social-media-container .content-container .copyright{
+    text-align: center;
+    font-size: 1.44rem;
+    font-weight: 300;
+}
+
+
+
+@media (min-width: 76.8em) {
+    .content-container {
+        width: 73.6em;
+        sbackground-color: red;
```

seems to be a typo here
```suggestion
        background-color: red;
```

---

I think Leroy uses this instead of commenting.